cdk/transactionAnalyzer/pom.xml

```diff
@@ -0,0 +1,100 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>software.amazon.event.kafkaconnector</groupId>
+    <artifactId>transactionAnalyzer</artifactId>
+    <version>1.0</version>
+    <packaging>jar</packaging>
+    <name>kafka-eventbridge-sink-transactionAnalyzer</name>
+    <url>https://github.com/awslabs/eventbridge-kafka-connector</url>
+    <description>Kafka consumer to process financial transactions</description>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <aws.schema.registry.version>1.1.15</aws.schema.registry.version>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
```

Comment:
Why not java 17 ?

Reply:
The repo uses `11` too, so IMHO it's ok to keep it consistent.